lib/Github/Api/PullRequest/Comments.php

```diff
@@ -38,13 +38,18 @@ public function configure($bodyType = null, $apiVersion = null)
         return $this;
     }
 
```

Comment:
Can you add a method doc block here? It would also be great if you linked to https://developer.github.com/v3/pulls/comments/#list-comments-in-a-repository